helpdesk/settings.py

```diff
@@ -25,6 +25,9 @@
 
 HAS_TAG_SUPPORT = False
 
+# Use international timezones
+USE_TZ: bool = True
```

Comment:
I don't understand why it's necessary to add this setting. It should already be set in the project django configuration using django-helpdesk.

Reply:
It can be changed in the project configuration but this would set it as the default value for new projects, so seems reasonable to me.

Reply:
The default value has changed in Django 4, this will prevent the deprecation warning from being emmiited.

Reply:
Alright, thank you for the explanation.